### ISSUE TITLE
Discover other walking entities that might also trample

### ISSUE THREAD

**Issue by matjojo** [matjojo/desire-paths#6]
- [x] Player
- [ ] Villager
- [ ] ???

**Comment by Arkhayla** [comment]
Conceptual Suggestion. If animals trampled grass, it would require players to build larger areas to store them in. Essentially, if you cram a bunch of sheep in to a small pen they would trample all their food. 

Just commenting for the sake of debate, perhaps this could be something to consider once a config can be implemented to add challenge if the player wants.

**Comment by matjojo** [comment]
Yeah that is the core config idea I had. Any animal that eats grass (Even modded ones if made smartly) ca eat my trampled blocks, so that should not be an issue.

The only scare I have is that it will lag a bit, it should not, but still I don't like the idea.

I will have to investigate if there are any other mobs that have nice set walking schedules. But I don't think so yet.
